### ISSUE TITLE
peerDependency issue npm3+

### ISSUE THREAD

**Issue by zslabs** [lazd/gulp-declare#9]
Running into an issue with one of the deps this lib depends on, inside `vinyl-maps`. Looks like there is a PR open to potentially fix it https://github.com/hughsk/vinyl-map/pull/11 but until that time, this lib doesn't appear to be working correctly with npm3+. Any suggestions or workarounds you'd recommend?

**Comment by lazd** [comment]
Yeah looks like we have to wait for that to be merged.

**Comment by coryzibell** [comment]
@lazd Would it be possible to switch the dependency to https://github.com/yoshokatana/vinyl-map2 ? That one is more actively maintained.

**Comment by lazd** [comment]
@coryzibell yeah there are a few ways to remedy this. I'd accept a pull request to replace `vinyl-map` with a more basic way of handling the stream, but I'd avoid switching to a fork of `vinyl-map` unless @hughsk is MIA as that fragments the community.

**Comment by benallfree** [comment]
FYI installing the peer dependency works in the mean time.

`npm install stream-browserify --save-dev`

You can also use `npm shrinkwrap` and then edit the `vinyl-map` dependency directly, but `shrinkwrap` won't work until the peer dependency can be resolved. Kind of a catch 22.
